src/model/SizingSystem.cpp

```diff
@@ -916,11 +956,18 @@ namespace model {
       return result;
     }
 
+    boost::optional<double> SizingSystem_Impl::autosizedOccupantDiversity() const {
+      boost::optional<double> result;
+      // TODO
```

Comment:
*  [ ] **TODO**

Reply:
I'm not totally sure where/how to retrieve these value(s). I added some notes for where I found anything "occupant diversity" related in the eplusout sql file.

Reply:
https://github.com/NREL/OpenStudio/issues/4446